stable/traefik/templates/NOTES.txt

```diff
@@ -11,3 +12,21 @@
          $ kubectl describe svc {{ template "fullname" . }} --namespace {{ .Release.Namespace }} | grep Ingress | awk '{print $3}'
 
 2. Configure DNS records corresponding to Kubernetes ingress resources to point to the load balancer IP/hostname found in step 1
+{{- end }}
+{{- if eq .Values.serviceType "NodePort" }}
+{{- if (and (not (empty .Values.service.nodePorts.https)) (not (empty .Values.service.nodePorts.http)))}}
+1. Traefik is listening on the following ports on the host machine:
+
+     http - {{ .Values.service.nodePorts.http }}
+     https - {{ .Values.service.nodePorts.https }}
+{{- else }}
+1. Traefik has been started and is binding to dynamically assigned ports on machine.
+
+     You can see the nodePorts which the service is using by running:
+
+          $ kubectl describe svc {{ template "fullname" . }} --namespace {{ .Release.Namespace }}
+
+{{- end }}
+
```

Comment:
There's an edge case here:

One, but not both, of `.Values.service.nodePorts.http` and `.Values.service.nodePorts.https` is set (non-empty). This would result in the `if` condition not being satisfied and the `else` would be triggered instead. The `else` says "Traefik has been started and is binding to dynamically assigned ports on machine." But that is true for only one of the ports.

I would suggest leaving the logic as it is, but changing the wording of the output under the `else` so that it doesn't say anything about dynamic binding, but just says (I'm paraphrasing) "here's how you can find the port numbers."

Reply:
@krancour I have taken care of your concerns and also resolved the merge conflict.

Reply:
There seems to be some linting errors in this PR. I will fix them and update